src/theme/styles/colours.ts

```diff
@@ -100,7 +104,7 @@ export const LIGHT_COLOURS = {
   labelBgOpen: '#77838F1A',
 
   // table & borders
-  borderPrimary: 'rgba(151, 151, 184, 0.3)',
+  borderPrimary: '#3A3A48',
```

Comment:
Removing alpha, as I realized the transparency might possibly have future color/blending interference with other elements.